.github/workflows/build-book.yaml

```diff
@@ -23,29 +23,15 @@ jobs:
 
       - name: Setup R
         uses: r-lib/actions/setup-r@master
-        with:
```

Comment:
I am almost certain this didn't help the original problem, since it only sets the option for that one action

Reply:
<img width="710" alt="Screen Shot 2020-10-06 at 3 20 32 PM" src="https://user-images.githubusercontent.com/599454/95266184-8c713000-07e7-11eb-82a8-b88c347b08d1.png">

Nope. Screenshot of the successful deployment after I bumped the bookdown cache.

Reply:
But how does it work? Did @jimhester add something specific to the setup-R action?

Reply:
Yes, that is my impression:

https://github.com/r-lib/actions/blob/79b0ee591d1cdce07ab0ff6a2519af150bd4390b/setup-r/action.yml#L14

And that makes its way into options set via an `.Rprofile`:

https://github.com/r-lib/actions/blob/79b0ee591d1cdce07ab0ff6a2519af150bd4390b/setup-r/src/installer.ts#L365

https://github.com/r-lib/actions/blob/79b0ee591d1cdce07ab0ff6a2519af150bd4390b/setup-r/lib/installer.js#L359

I added this originally based on research on the action and what others are doing in similar bookdown projects.

Reply:
Ah, it's a lack of documentation. I've filed an issue.

Reply:
Thinking about this more, I do think the right place for this setting is close to the book generation, since the root cause is that bookdown doesn't handle ANSI escapes. (But that's something that I will hopefully fix in the next month, rendering the point moot)